src/cozmo/robot.py

```diff
@@ -725,7 +725,7 @@ def abort_all_actions(self):
         Abort / Cancel any action that is currently either running or queued within the engine
         '''
         # RobotActionType.UNKNOWN is a wildcard that matches all actions when cancelling.
-        msg = _clad_to_engine_iface.CancelAction(robotId=self.robot_id,
+        msg = _clad_to_engine_iface.CancelAction(robotID=self.robot_id,
```

Comment:
No, unfortunately it's not completely consistent in the clad files - SetRobotVolume is declared as:
```message SetRobotVolume {
  uint_8   robotId,
  float_32 volume
}```